specification/containerservice/resource-manager/Microsoft.ContainerService/fleet/readme.python.md

```diff
@@ -0,0 +1,59 @@
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+
+```yaml $(python)
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+package-name: azure-mgmt-containerservicefleet
```

Comment:
New `readme.python` for fleet, modified `package-name` from `azure-mgmt-containerservice` to `azure-mgmt-containerservicefleet`.